src/ga4gh/vrs/extras/translator.py

```diff
@@ -11,6 +11,8 @@
 from collections.abc import Mapping
 from typing import Protocol
 
+from typing_extensions import deprecated
```

Comment:
if we're importing it by name then it should probably be added as an explicit dependency to pyproject.toml